app/controllers/report_controller/schedules.rb

```diff
@@ -241,7 +241,7 @@ def schedule_edit
       add_flash(_("All changes have been reset"), :warning) if params[:button] == "reset"
       if x_active_tree != :reports_tree
         # dont set these if new schedule is being added from a report show screen
-        obj = find_checked_ids_with_rbac(MiqSchedule)
+        obj = find_records_with_rbac(MiqSchedule, checked_or_params)
         obj[0] = params[:id] if obj.blank? && params[:id]
```

Comment:
The method `find_records_with_rbac` always returns something (see https://github.com/ManageIQ/manageiq-ui-classic/pull/3131), so the test `obj.blank?` will always be `false`, therefore it can be removed.